public/components/event_analytics/explorer/datasources/datasources_selection.tsx

```diff
@@ -190,7 +189,7 @@ export const DataSourceSelection = ({ tabId }: { tabId: string }) => {
    * Subscribe to data source updates and manage the active data sources state.
    */
   useEffect(() => {
-    const subscription = dataSources.dataSourceService
+    const subscription = dataSources?.dataSourceService
```

Comment:
Could you give me more details about this change?

Reply:
@sumukhswamy could you check if this is one of the scenario that was introduced by the MDS change?